src/addon.ts

```diff
@@ -76,85 +75,77 @@ function checkDllExports(filepath: string): boolean {
 }
 
 export function createReleaseFromDll(
```

Comment:
Removed the addon name from error messages so this function doesn't need the `addon` object anymore.
Also renamed a few variables to use shorthand properties when returning the release.